api/v2/clairpb/clair.proto

```diff
@@ -0,0 +1,133 @@
+syntax = "proto3";
+option go_package = "clairpb";
+
+package clairpb;
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+
+message Vulnerability{
```

Comment:
Change this so that you always have the space in between the message name and the curly bracket `message Name {` instead of `message Name{`